tools/storage-scale-driver-snap.sh

```diff
@@ -162,8 +198,9 @@ function get_kind()
 function help()
 {
    # Display Help
-   echo "USAGE: storage-scale-driver-snap.sh [-n|o|p|s|v|h]"
+   echo "USAGE: storage-scale-driver-snap.sh [-l|n|o|p|s|v|h]"
    echo "options:"
+   echo "     l     Collect logs only from driver pods which are running along with sidecars"
```

Comment:
Can we rewrite this log message such as  `Collect logs from driver pods running on same node as sidecar pods`

Reply:
Done!